caffe2/python/operator_test/conv_transpose_test.py

```diff
@@ -37,6 +38,11 @@ def test_convolution_transpose_layout_legacy_args(
         b = np.random.rand(output_channels).astype(np.float32) - 0.5
         outputs = {}
         for order in ["NCHW", "NHWC"]:
+            # NHWC not supported in MIOpen, run HIP
```

Comment:
I think you can just add one line here:
```
assume(order == 'NCHW' or not (hiputl.run_in_hip(gc, dc) and engine == 'CUDNN'))
```
same for the other places in this file.